docs/guides/cloud.rst

```diff
@@ -77,32 +77,59 @@ If you'd prefer, you can also manage your account via `the EdgeDB Cloud
 web-based GUI <https://cloud.edgedb.com/>`_.
 
 The first time you access the web UI, you will be prompted to log in. Once you
-log in with your GitHub account, you'll see your instance list.
+log in with your account, you'll be on the Instances tab of the front page
+which shows your instance list. The other two tabs allow you to manage your
+Org Settings, and Billing.
+
+Instances
+---------
 
 If this is your first time accessing EdgeDB Cloud, this list will be empty. To
-create an instance, click "Create new instance." This will pop up a modal
+create an instance, click "Create new instance". This will pop up a modal
```

Comment:
Period inside the quotation is the prevailing grammatical convention in the US. Since I don't think it will create any confusion here, I'd vote for sticking to it (even though it honestly really bugs me).
```suggestion
create an instance, click "Create new instance." This will pop up a modal
```

Reply:
Ew. But okay, we'll follow that convention's lead I guess.